source/includes/stackpath/_workloads.md

```diff
@@ -304,8 +307,14 @@ Required | &nbsp;
  
  Optional | &nbsp;
  ------- | -----------
+  `slug`<br/>*string* | A workload's programmatic name. Workload slugs are used to build its instances names. If not provided, defaults to workload's name.
  `addAnyCastIpAddress`<br/>*boolean* | Option to AnyCast IP Address.
- `slug`<br/>*string* | A workload's programmatic name. Workload slugs are used to build its instances names. If not provided, defaults to workload's name.
+  `publicPort`<br/>*string* | A single port, such as 80 or a port range, such as 1024-65535 for which a network policy rule will be created for the workload.
+ `publicPortSrc`<br/>*string* | A subnet that will define all the IPs allowed by the network policy rule.
+ `publicPortDesc`<br/>*string* | A summary of what the network policy rule does or a name for it. It is highly recommended to give a unique description to easily identify a network policy rule.
+ `protocol`<br/>*string* | Protocol for the network policy rule. Supported protocols are: `TCP`, `UDP` and `TCP_UDP`.
+ `firstBootSshKey`<br/>*string* | The ssh key(s) for the VM image. Keys are delimited by a newline, `\n`. Only applicable to workloads of `type` 'VM'.
```

Comment:
We have this field in the required section as well as in the optional section. Shouldn't this be only in one section?

Reply:
good catch

Reply:
01d4d5d